Library/Homebrew/utils/gems.rb

```diff
@@ -151,6 +154,33 @@ def install_bundler!
     ENV["BUNDLER_VERSION"] = old_bundler_version
   end
 
+  def user_gem_groups
+    @user_gem_groups ||= if File.exist?(GEMGROUPS_FILE)
+      File.readlines(GEMGROUPS_FILE, chomp: true)
+    else
+      # Backwards compatibility
```

Comment:
Could this get a commented-out `odeprecated` or something so we don't need to carry it forever?

Reply:
I'll leave comment to just outright remove it after (how many?) months. If the user hasn't had a bundler install run in that period, then they're likely no longer needing the remembered gems. And even if so, all that happens is the user will need to install them again, which they probably do anyway if it's been several months.

Reply:
As far as I can gather is it is [configured](https://github.com/Homebrew/brew/blob/51889e7a089f5aad0c3c4f21f0dbd3fc382f0e08/.vscode/settings.json#L10) to run https://github.com/Homebrew/brew/blob/master/Library/Homebrew/shims/gems/rubocop which runs https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/rubocop.sh which runs `brew install-bundler-gems` anyway?

Reply:
Let's say ~1 month and can you create a `help wanted` issue noting that date so we don't forget?